allure-app-under-test/src/main/java/com/capgemini/ntc/webapi/endpoint/soap/FarenheitToCelsiusMethod_Request_FromCode.java

```diff
@@ -39,7 +39,7 @@ public FarenheitToCelsiusMethod_Request_FromCode() {
 	/**
 	 * @return Generate SOAP response in String format
 	 */
-	public String getMessage() {
+	public String getEndpoint() {
```

Comment:
This method "getEndpoint" change does not correlate with method core funtion. Function returns request message  and not endpoint.

Reply:
Ok, I have change it back to getMessage()

**Problem to consider:** Now *IWebApi* include only one method - *getEnpoint()*. For each SOAP page it will need to be implemented even if we will do not use it. 

**Propositions:** 
1. Create 2 Interfaces - one for SOAP, one for REST pages
2. Remove *IWebApi*
3. Unify the name of the method for both cases in *IWebApi* (I wanted to achieve that with getEndpoint name)